client/src/send_and_confirm_transactions_in_parallel.rs

```diff
@@ -31,7 +31,7 @@ use {
     tokio::{sync::RwLock, task::JoinHandle, time::Instant},
 };
 
-const BLOCKHASH_REFRESH_RATE: Duration = Duration::from_secs(10);
+const BLOCKHASH_REFRESH_RATE: Duration = Duration::from_secs(5);
```

Comment:
shouldn't it be 150 slots (`MAX_PROCESSING_AGE`)?

Reply:
This number is used for refreshing both the block height and the blockhash on a regular cadence. So while the blockhash is definitely valid for `MAX_PROCESSING_AGE`, we also need to refresh the block height regularly to see if the blockhash is still valid